### ISSUE TITLE
ListView onClick listener not called

### ISSUE THREAD

**Issue by anthony-skr** [arcadefire/nice-spinner#4]
Hello,

I noticed the ListView onClick listener is not working on pre-Lollipop devices.

Checked on Internet, found this : http://stackoverflow.com/a/11244139/2743222

I replaced PopupWindow construction by this : 

```java
mPopup = new PopupWindow(mListView,
                ViewGroup.LayoutParams.WRAP_CONTENT,
                ViewGroup.LayoutParams.WRAP_CONTENT, true);
```

Works like a charm :+1:

Anthony.

**Comment by arcadefire** [comment]
Hey @anthony-skr, thank you for sharing! I'll up a fix asap!

**Comment by arcadefire** [comment]
This has been fixed in 7859190131248f2248b15762c050f669c6754add, thank you!
